vsts/pipelines/_signbinary.yml

```diff
@@ -0,0 +1,63 @@
+steps:
+- powershell: 'Write-Host "Running Sign Oryx binaries"'
+  displayName: 'Show Build Phase'
+
+- powershell: |
+   Write-Host "##vso[task.setvariable variable=SignType]real"
+   Write-Host "##vso[task.setvariable variable=AGENTBUILD]true"
+  displayName: 'Set sign binary variable'
+  condition: and(in(variables['Build.Reason'], 'Schedule', 'Manual'), eq(variables['Build.DefinitionName'], 'Oryx-CI'))
```

Comment:
we should avoid repeating these checks in every task. Add a new task that checks those things, and create a variable, e.g. "SignBinary" = "true" if they pass

Reply:
yes I saw it … I already have a task where I'm setting signtype variable so from next task onward I will just use it